src/main/java/Duke.java

```diff
@@ -1,10 +1,24 @@
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreeting();
+        Command.filterInput();
+        printExit();
     }
+
+    public static void textBorder(String displayText) {
```

Comment:
Methods should be verbs.Can consider a more reasonable name, such as printBetweenLines, to better explain what the method does.

Reply:
Thanks for your feedback on this method! Managed to rename this method with a more appropriate name.